### DISCUSSION TITLE
Rename "Software Updates" to "Packages Updates"?

### DISCUSSION THREAD

**Discussion by siewers** [Devolutions/UniGetUI#2901] [General]
I tried searching for something about this, but I couldn't find anything.

I was wondering why the tabs are all named something about "packages" except for updates, where it's referred to as "software":
- Discover _**Packages**_
- **_Software_** Updates
- Installed **_Packages_**
- **_Package_** Bundles

Wouldn't it be more consistent to rename the tab "Software Updates" to "Package Updates"?

**Comment by marticliment** [comment]
I will investigate

**Comment by siewers** [comment]
It even say "x number of packages can be updated" in the notification message.

**Comment by Anders0lesen** [comment]
Or even remove packages entirely and call it just:
 -> Discover
 -> Update (Show the number of packages that could be updated as a notification in the top right of the Icon?)
 -> Installed
 -> Bundle

**Reply by jerone** [reply]
Agree. One word makes it also easier to read, then multiple lines.

**Comment by jerone** [comment]
Related issue: https://github.com/marticliment/UniGetUI/issues/2990
